app/controllers/api/api_application_controller.rb

```diff
@@ -21,6 +22,19 @@ def record_not_found
     status = :not_found
     code = 404
     message = 'Record_not_found'
-    render json: { status:, code:, message: }, status:
+
+    render_error(code, message, status)
+  end
+
+  def handle_record_invalid_error
+    code = 400
```

Comment:
I understand the need to handle this error, but since this method isn't used in this PR, I think it should be on a different PR, along with the places it is called

Reply:
alright, I removed that commit for now